stock_reserve_rule/models/stock_reserve_rule.py

```diff
@@ -140,6 +176,44 @@ class StockReserveRuleRemoval(models.Model):
         "When empty, any packaging can be removed.",
     )
 
+    missing_reserve_rule = fields.Boolean(
```

Comment:
Do you have details of the expectations for this behavior (some specs?), not sure to get it

Reply:
My last comment on the related card (that I just added) following a meet with Joël :)